### DISCUSSION TITLE
2023-09-27

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#586] [daily_meeting]
### 개인 일정
- 노아 5 - 6시 근로
- 룩소 4 - 5시 근로

### 오늘의 할 일
- 프론트
  - 반응형 마무리
  - 개별 태스크 진행
- 백엔드
  - 서버 이중화 인프라 작업 진행
- 전체
  - 인프런, 스프 홍보 진행

### 내일의 할 일
- 혼자하기 플로우? 튜토리얼 스터디?

**Comment by aak2075** [comment]
### 체크인

10점 - 불타는 수요일이네요

### 체크아웃

10점 - 항해 해야되나요

**Comment by woo-jk** [comment]
### 체크인
10점 - 내일부터 긴 연휴네요 ㅎ

### 체크아웃
10점 - 즐거운 한가위~!

**Comment by jaehee329** [comment]
### 체크인
9점 - 오늘만 지나면 휴일이네요 ㅠㅠ 조금 설레서 -1점

### 체크아웃
10점 - 즐거운 추석~

**Comment by nlom0218** [comment]
### 체크인

10점 - 달달구리한거 좋아요

### 체크아웃

10점 - 즐추!

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 알고리즘 공부를 시작하기로 마음 먹었는데 오늘 아침 스터디에서 잘 해낸 것 같아서 기분이 좋네용

### 체크아웃
10점 - 오늘 많은 걸 했네요! 좋습니당

**Comment by woosung1223** [comment]
### 체크인
8점 - 내일부터 연휴이군요

### 체크아웃
9점 - 오늘 그래도 많이 한 것 같아서 좋아요

**Comment by yeopto** [comment]
### 체크인

8점 - 진짜 오늘은 역대급 지옥철이었습니다.

### 체크아웃

9점 - 휴일이라 10점이지만 할게 많아서 -1
